tests/test_widgets.py

```diff
@@ -3530,6 +3534,62 @@ def test_textbox_autoscroll(self):
             "|                                      |\n" +
             "+--------------------------------------+\n")
 
+    def test_layout_gutters(self):
+        """
+        Validate that the gutters parameter works for Layouts
+        """
+        # Now set up the Frame ready for testing
+        screen = MagicMock(spec=Screen, colours=8, unicode_aware=False)
+        scene = Scene([], duration=-1)
+        canvas = Canvas(screen, 10, 40, 0, 0)
+        form = Frame(canvas, canvas.height, canvas.width)
+        layout = Layout([18, 4, 18], gutter=3)
```

Comment:
Are you deliberately using numbers that will hit rounding errors?

Reply:
Been a while, but IIRC, this doesn't cause the rounding problem. I vaguely recall having a hard time to get it to happen after I'd built the gap feature. I had been playing around with rather wide screens, and I suspect it only comes up when you've got absolutely large layouts.